src/components/visualizations/chart/test/ChartTransformation.spec.tsx

```diff
@@ -472,4 +474,75 @@ describe("ChartTransformation", () => {
             });
         },
     );
+
+    describe("axis labels alignment on dual bar chart", () => {
+        function createComponent(chartConfig: IChartConfig) {
+            const props = {
+                ...fixtures.barChartWith2MetricsAndViewByAttribute,
+                config: {
+                    type: VisualizationTypes.BAR,
+                    ...chartConfig,
+                },
+                onDataTooLarge: noop,
+            };
+            return mount(
+                <IntlWrapper>
+                    <ChartTransformation {...props} />
+                </IntlWrapper>,
+            );
+        }
+
+        it("should align secondary Y axis labels to left", () => {
+            const chartConfig: IChartConfig = {
+                secondary_xaxis: {
+                    measures: ["wonMetric"],
+                    rotation: "90",
+                },
+            };
+            const wrapper = createComponent(chartConfig);
+            const chartConfigProps = wrapper.find(Chart).prop("config");
+            const label = get(chartConfigProps, "yAxis.1.labels");
+
+            expect(label.align).toBe("left");
+            expect(label.y).toBe(undefined);
+        });
+
+        it("should align Y axis label to right and secondary Y axis labels to left", () => {
+            const chartConfig: IChartConfig = {
+                xaxis: {
+                    rotation: "90",
+                },
+                secondary_xaxis: {
+                    measures: ["wonMetric"],
+                    rotation: "90",
+                },
+            };
+            const wrapper = createComponent(chartConfig);
+            const chartConfigProps = wrapper.find(Chart).prop("config");
+
+            const labels = get(chartConfigProps, "yAxis.0.labels");
+            expect(labels.align).toBe("right");
+            expect(labels.y).toBe(8);
+
+            const secondaryLabels = get(chartConfigProps, "yAxis.1.labels");
+            expect(secondaryLabels.align).toBe("left");
+            expect(secondaryLabels.y).toBe(undefined);
+        });
+
+        it("should not align secondary Y axis labels to left on other charts", () => {
+            const chartConfig: IChartConfig = {
+                type: VisualizationTypes.COLUMN,
+                secondary_yaxis: {
+                    measures: ["wonMetric"],
+                    rotation: "90",
+                },
+            };
+            const wrapper = createComponent(chartConfig);
+            const chartConfigProps = wrapper.find(Chart).prop("config");
+            const label = get(chartConfigProps, "yAxis.1.labels");
+
+            expect(label.align).toBe(undefined);
+            expect(label.y).toBe(undefined);
```

Comment:
use `toEqual` instead of `toBe`.
`expect(label).toBe({});`

same for other tests in this file

Reply:
I think `toEqual ` is for asserting object, and `toBe` is for primitive types.